runtime/gc_glue_java/MarkingSchemeRootClearer.cpp

```diff
@@ -325,7 +327,7 @@ MM_MarkingSchemeRootClearer::scanContinuationObjects(MM_EnvironmentBase *env)
 						while (NULL != object) {
 							gcEnv->_markJavaStats._continuationCandidates += 1;
 							omrobjectptr_t next = _extensions->accessBarrier->getContinuationLink(object);
-							if (_markingScheme->isMarked(object)) {
+							if (_markingScheme->isMarked(object) && !VM_VMHelpers::isContinuationFinished((J9VMThread *)env->getLanguageVMThread(), object)) {
```

Comment:
why did you use this helper - just for being concise?
we already used VM_ContinuationHelpers::isFinished() directly in MM_GCExtensions::releaseNativesForContinuationObject